### PR TITLE
renaming dal lib

### PR DESCRIPTION
**Pull request by napetrov**
in this change we rename daal_sycl to onedal_sycl starting from Beta08

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -103,11 +103,14 @@
 
 if dpcpp:
     DPCPP_CFLAGS = ['-D_DPCPP_']
-    DPCPP_LIBS = ['OpenCL', 'sycl', 'daal_sycl']
+    DPCPP_LIBS = ['OpenCL', 'sycl', 'onedal_sycl']
     if IS_LIN:
         DPCPP_LIBDIRS = [jp(dpcpp_root, 'linux', 'lib')]
     elif IS_WIN:
         DPCPP_LIBDIRS = [jp(dpcpp_root, 'windows', 'lib')]
+    if dal_build_version == (2021,7):
```
Comment (Alexander-Makaryev) [inline setup.py:111]: maybe
dal_build_version  <= (2021,7)
?

Comment (napetrov) [reply setup.py:111]: I tough about this - but not sure how  well  this work. 

in general all previous beta versions are obsolete and this code will be removed. together with previous versions

Comment (napetrov) [reply setup.py:111]: so taking in mind time constraints - i would go with safe option )

**Pull request merged**